packages/core/src/login/oidc/IIssuerConfig.ts

```diff
@@ -67,4 +67,5 @@ export interface IIssuerConfig {
   requireRequestUriRegistration?: boolean;
   opPolicyUri?: string;
   opTosUri?: string;
+  solidOidcSupported?: boolean;
```

Comment:
Hmm, am I misreading the spec if I think this should be:

```suggestion
  solidOidcSupported?: "https://solidproject.org/TR/solid-oidc";
```

?

Reply:
Yeah, it's definitely a good point - I'm assuming Nic is making the assumption that the presence of *any* value means that Solid OIDC is supported by the IdP, and the particular value isn't relevant (the Solid OIDC spec could make that clearer perhaps, by explicitly stating that a IdP MUST NOT provide any value at all if they don't support it (i.e., stating "solid_oidc_supported: "Not supported"" would be non-conformant). I'd be Ok with that, but I would definitely add a comment explaining that. I created a spec issue [here](https://github.com/solid/specification/issues/249).

Reply:
You'll want to verify that the _value_ is equal to that particular URL, not just that the key exists.

Reply:
You're absolutely right, that's actually why I initially made it a string, and somehow I was jumping aound this morning and forgot about that \o/

Reply:
Fixed in https://github.com/inrupt/solid-client-authn-js/pull/1154/commits/cbc5de1f922b34b675a070e88122067704bfc537